### PR TITLE
Release v1.19.0

### PR DESCRIPTION
**Pull request by theishshah**

### CODE REVIEW

File: changelog/generated/v1.19.0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- Reverted removal of  `--skip-tls` flag and mark the flag as deprecated. ([#5633](https://github.com/operator-framework/operator-sdk/pull/5633))
+
+### Changes
+
+- scorecard will now printout test results, if any, even when there is a context timeout. This will allow underlying errors from a test to be shown helping users address the issues. ([#5628](https://github.com/operator-framework/operator-sdk/pull/5628))
+
+### Bug Fixes
+
+- (java/v1alpha) bumping the java-operator-plugins dependency to v0.3.0. This release includes the following items.
+### Bug Fixes - chore: update to JOSDK extension 3.0.4 and Quarkus 2.7.3 (Fixes issue 74). ([#5542](https://github.com/operator-framework/operator-sdk/pull/5542))
```
Comment (jmrodri) [inline changelog/generated/v1.19.0.md:18]: We should update this. 
```suggestion

- (java/v1alpha) bumped the java-operator-plugins dependency to [v0.3.0](https://github.com/operator-framework/java-operator-plugins/releases/tag/v0.3.0). ([#5588](https://github.com/operator-framework/operator-sdk/pull/5588))
```

**Review by jmrodri — CHANGES_REQUESTED** [review]
Minor change to change log.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Pull request merged**